tests/testthat/setup.R

```diff
@@ -0,0 +1,13 @@
+library(mockery)
```

Comment:
I think we do not need this? `with_mocked_bindings` provided by `testthat`.

Reply:
Can you move these helper functions into `helper-hintr.R`. Functions you want to source before running tests are in `helper-` by convention and then testthat will source them as part of its setup.

Reply:
Ah I see using this library to remove the namespacing when creating a mock. In general I always use namespaced calls in R packages as they affect the users environment which can cause problems. This is fine here as just sourced in tests, but perhaps worth moving into `tests/testthat.R` with the other library calls.

Reply:
okay makes sense! will move the using library bit to `test/testthat.R` and helper functions to `helper-hintr.R`!